modules/virt-preventing-workload-updates-during-eus-update.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+:_content-type: PROCEDURE
+[id="virt-preventing-workload-updates-during-eus-update_{context}"]
+= Preventing workload updates during an EUS-to-EUS update
+
+When you update from one Extended Update Support (EUS) version to the next, you must manually disable automatic workload updates to prevent {VirtProductName} from migrating or evicting workloads during the update process.
+
+.Prerequisites
+
+* You are running an EUS version of {product-title} and want to update to the next EUS version. You have not yet updated to the odd-numbered version in between.
+
+* You read "Preparing to perform an EUS-to-EUS update" and learned the caveats and requirements that pertain to your {product-title} cluster.
+
+* You paused the worker nodes' machine config pools as directed by the {product-title} documentation.
+
+* Your {VirtProductName} subscription uses the *Automatic* approval strategy.
++
+[NOTE]
+====
+If you use the *Manual* approval strategy, you must approve all pending updates in the web console. This is discouraged because your cluster becomes unsupported if your {VirtProductName} and {product-title} versions are out of sync. For more details, refer to the "Manually approving a pending Operator update" section.
```

Comment:
This is misleading. And I don't see how manual approval can lead to `cluster becomes unsupported if your {VirtProductName} and {product-title} versions are out of sync`